linebot/filter.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2020 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package linebot
+
+import "encoding/json"
+
+// Filter interface
+type Filter interface {
+	Filter()
```

Comment:
It seems Filter() method do nothing, just represents the struct is one of filter.
If so, I think it is better to write to comment, and IMHO, the better method signature is:
```
IsFilter() bool
```
and each implementation of this method should return true.
With just `Filter()`, it seems we can filtering with the method.
HDYT?

Reply:
Hi, I think it's ok to change to IsFilter(), however the intention is to comply with the naming rule, i.e. 
`// Message interface
type Message interface {
    Message()
}`

I have no objection on this. Just need a decision :)

Reply:
ah.. sorry, I haven't see that interface, your naming is right.
Thanks,